db/migrate/20170705213845_change_type_and_name_of_games_status_id.rb

```diff
@@ -1,6 +1,6 @@
 class ChangeTypeAndNameOfGamesStatusId < ActiveRecord::Migration[5.0]
   def change
     rename_column :games, :game_status_id, :game_status
-    change_column :games, :game_status, :string
+    change_column :games, :game_status, :string, default: 'Open'
```

Comment:
We'll need to define `game_status` options before we get deeper into the project.